src/axom/spin/UniformGrid.hpp

```diff
@@ -726,13 +711,10 @@ void UniformGrid<T, NDIMS, ExecSpace, StoragePolicy>::getCandidatesAsArray(
     // Step 2: exclusive scan for offsets in candidate array
     // Intel oneAPI compiler segfaults with OpenMP RAJA scan
   #ifdef __INTEL_LLVM_COMPILER
-  using exec_policy = typename axom::execution_space<axom::SEQ_EXEC>::loop_policy;
+  axom::exclusive_scan<axom::SEQ_EXEC>(outCounts, outOffsets);
   #else
-  using exec_policy = typename axom::execution_space<ExecSpace>::loop_policy;
+  axom::exclusive_scan<ExecSpace>(outCounts, outOffsets);
   #endif
```

Comment:
Note: same logic here about a bugfix for scans w/ oneAPI

Reply:
I removed this logic since it was consolidated into ``axom::exclusive_scan``.